x/smart-account/authenticator/composite.go

```diff
@@ -56,6 +56,9 @@ func onSubAuthenticatorsAdded(ctx sdk.Context, account sdk.AccAddress, data []by
 	subAuthenticatorCount := 0
 	for id, initData := range initDatas {
 		authenticatorCode := am.GetAuthenticatorByType(initData.Type)
+		if authenticatorCode == nil {
+			return errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, "sub-authenticator `OnAuthenticatorAdded` type not registered in manager")
```

Comment:
the error descriptions seem off. Makes it sound like the type of authenticator is `OnAuthenticatorAdded`